### ISSUE TITLE
[0781 PS] (3) Review Page 'Completion Choice' Skeleton

### ISSUE THREAD

**Issue by SamStuckey** [department-of-veterans-affairs/va.gov-team#97068]
## SOFT BLOCKED BY DESIGN (see comments below)

We need to have our form JSON boiler plate in place before this work can be _completed_.

- department-of-veterans-affairs/va.gov-team#97063 

### Description

Entry Point: 3.17

Exit Point: 3.1c, 3.1b

Add the basic navigation and 'completion choice' for Review page

#### OUT OF SCOPE

- Form review section dropdowns. There is a much larger discussion yet to happen around these. This will happen in department-of-veterans-affairs/va.gov-team#97089 

### Acceptance Criteria

- As a user visiting 3.17 I see a placeholder for form section drop downs
- An option to mark form as complete
    - YES and Continue 
        - routes to 3.1c
        - This sets my forms status as 'complete'
    - NO and Continue 
        - routes to 3.1b

If the user selects the "Yes, I’ve completed this additional form" direct them to the 0781 Completed Return to  [3.1c] Start page - complete status
If the user selects the "No, I’ll come back to it later OR no response" direct them to the 0781 Completed Return to  [3.1b] Start page - in progress status 

### Expected Deployments

- FE: 
    - deploy the page elements that handle routing and set the form status

[See the Planning Doc for more info](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/engineering_research/0781_paper_sync_plannig.md)

**Comment by kylesoskin** [comment]
Changing to ready state, if needed, unanswered questions can be refined into a new ticket.
